### PR TITLE
DOC: Example for integrate.simps

### PR DESCRIPTION
**Pull request by peterSweter**
I have created a few simple examples for integrate.simps

### CODE REVIEW

File: scipy/integrate/quadrature.py [hunk 1]
```diff
@@ -380,6 +380,28 @@ def simps(y, x=None, dx=1, axis=-1, even='avg'):
     the samples are not equally spaced, then the result is exact only
     if the function is a polynomial of order 2 or less.
 
+    Examples
+    --------
+    >>> import numpy as np
```
Comment (person142) [inline scipy/integrate/quadrature.py]: Don't need to include the NumPy import.

File: scipy/integrate/quadrature.py [hunk 2]
```diff
@@ -380,6 +380,28 @@ def simps(y, x=None, dx=1, axis=-1, even='avg'):
...
+    40.5
+
+    >>> y = np.power(x, 3)
+    >>> integrate.simps(y,x)
+    1642.5
+
+    >>> integrate.simps(y, x, even='first')
+    1644.5
+
+
```
Comment (person142) [inline scipy/integrate/quadrature.py]: Ended up with too many blank lines here.

File: scipy/integrate/quadrature.py [hunk 3]
```diff
@@ -380,6 +380,28 @@ def simps(y, x=None, dx=1, axis=-1, even='avg'):
...
+    >>> import numpy as np
+    >>> from scipy import integrate
+    >>> x = np.arange(0,10)
+    >>> y = np.arange(0,10)
+
+    >>> integrate.simps(y,x)
+    40.5
+
+    >>> y = np.power(x, 3)
+    >>> integrate.simps(y,x)
```
Comment (person142) [inline scipy/integrate/quadrature.py]: Might be good to compare to the exact answer.

**Review by person142 — COMMENTED** [review]
Good idea; make sure to add spaces after commas (i.e. `np.arange(0, 10)`).

**Comment by andyfaff** [comment]
@person142, this looks mergeable to me, did you want to merge yourself?

**Comment by andyfaff** [comment]
Commits should probably be squashed.

**Comment by person142** [comment]
@andyfaff go for it. Squash and merge will be fine to condense things.

**Pull request merged**